src/killport.rs

```diff
@@ -133,9 +101,10 @@ impl KillportOperations for Killport {
 
             for process in target_processes {
                 // Check if the process name contains 'docker' and skip if in docker mode
-                if docker_present && process.name.to_lowercase().contains("docker") {
+                if docker_present && process.get_name().to_lowercase().contains("docker") {
```

Comment:
> Docker container killing is working, however it doesn't work properly if you don't explicitly specify the container mode (I think its trying to kill the wrong one causing it to fail?)

I think it is probably because it is killing the Docker process? This conditional here avoids killing the docker process if it is on "auto" mode.

Reply:
I'm pretty sure the strange behavior I was seeing here was due to the parent killing behavior, I believe it was trying to go up the tree and kill the parents of the container like WSL (Because they didn't meet the matching name)

Fixed this when I merged the parents onto the processes rather than in the list to kill, so not an issue anymore